src/middleware/company.authentication.middleware.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response, NextFunction } from 'express';
+import { logger } from '../utils/logger';
+import { authMiddleware, AuthOptions } from "@companieshouse/web-security-node";
+import { CHS_URL, OVERSEAS_ENTITY_PRESENTER_URL } from '../config';
+import { getApplicationData } from "../utils/application.data";
+import { ApplicationData } from "../model";
+import { EntityNumberKey } from "../model/data.types.model";
+
+export const companyAuthentication = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `COMPANY AUTHENTICATION REQUEST`);
```

Comment:
Any reason this output message is in capitals? It will make it stand out in the logs, but does it need to?

Reply:
@mwestacott , no reason to and can surely make it follow the existing format